### PR TITLE
Fix list index out of range

### PR DESCRIPTION
**Pull request by fernandog**
Fixes https://github.com/pymedusa/Medusa/issues/2043

@duramato

### CODE REVIEW

File: medusa/providers/torrent/html/bithdtv.py [hunk 1]
```diff
@@ -147,8 +147,8 @@ def parse(self, data, mode):
                     if not all([title, download_url]):
                         continue
 
-                    seeders = try_int(cells[8].get_text(strip=True))
-                    leechers = try_int(cells[9].get_text(strip=True))
+                    seeders = try_int(cells[8].get_text(strip=True)) if len(cells) > 7 else 1
```
Comment (medariox) [inline medusa/providers/torrent/html/bithdtv.py]: What you probably want is ```>= 8```

Comment (fernandog) [inline medusa/providers/torrent/html/bithdtv.py]: in math is the same result =)

Comment (medariox) [inline medusa/providers/torrent/html/bithdtv.py]: Hmm, not really. ```> 9``` would be...

Comment (fernandog) [inline medusa/providers/torrent/html/bithdtv.py]: so 8 > 7 is false? lol

Comment (medariox) [inline medusa/providers/torrent/html/bithdtv.py]: Just think about what you are doing can be right....

Comment (medariox) [inline medusa/providers/torrent/html/bithdtv.py]: It has nothing to do with Math. You have to think about what you are doing and what you want to do.

File: medusa/providers/torrent/html/bithdtv.py [hunk 2]
```diff
@@ -147,8 +147,8 @@ def parse(self, data, mode):
                     if not all([title, download_url]):
                         continue
 
-                    seeders = try_int(cells[8].get_text(strip=True))
-                    leechers = try_int(cells[9].get_text(strip=True))
+                    seeders = try_int(cells[8].get_text(strip=True)) if len(cells) > 9 else 1
```
Comment (medariox) [inline medusa/providers/torrent/html/bithdtv.py]: It's still wrong 😢

**Pull request merged**